dashboards/jupyterhub.jsonnet

```diff
@@ -30,9 +30,58 @@ local templates = [
   ),
 ];
 
+/*
+ * Prometheus query for selecting pods
+ *
+ * @name componentLabel
+ *
+ * @param component The component being selected
+ * @param cmp (default `'='`) The comparison to use for the component label, e.g. for exclusions.
+ *
+ * @return prometheus query string selecting pods for a certain component
+ */
+local componentLabel(component, cmp='=') =
```

Comment:
Let's put functions in a separate jsonnet file and import them? Maybe we can reuse them in `cluster.jsonnet` too that way?

Reply:
Now I get to learn how imports work!